### PR TITLE
chore: cleanup API domain docs

### LINKED ISSUE #825: Documentation for Post

### PR DESCRIPTION
**Pull request by mildwonkey**
Remember that pull request I said I opened yesterday? Here it is. API domain docs clean up. (I swear I did this already)

Closes #825

### CODE REVIEW

File: docs/reference/domains/api-domain.md [hunk 1]
```diff
@@ -26,23 +29,23 @@ domain:
...
         url: "https://example.com/health/ready"
-        # method (optional): The HTTP Method to use for the API call. "get" and "post" are supported. Default is "get".
+        # method (optional, default get): The HTTP Method to use for the API call. "get" and "post" are supported. Default is "get".
         method: "get"
         # parameters (optional): parameters to append to the URL. Lula also supports full URIs in the URL.
         parameters: 
           key: "value"
         # Body (optional): a json-compatible string to pass into the request as the request body.
         body: |
 stringjsondata
```
Comment (CloudBeard) [inline docs/reference/domains/api-domain.md:39]: Does this need indented?

Comment (mildwonkey) [reply docs/reference/domains/api-domain.md:39]: Nope, it's a string literal after the pipe (`|`) so any whitespace could be considered part of that string.

**Review by meganwolf0 — APPROVED** [review]

**Review by CloudBeard — APPROVED** [review]

File: docs/reference/domains/api-domain.md [hunk 2]
```diff
@@ -52,7 +55,7 @@ stringjsondata
 
 ## API Domain Resources
 
-The API response body is serialized into a json object with the Request's `Name` as the top-level key. The API status code is included in the output domain resources under `status`. `raw` contains the entire API repsonse in an unmarshalled (`json.RawMessage`) format.
+The API response body is serialized into a json object with the `request` `name` as the top-level key. The API status code is included in the output domain resources under `status`. `raw` contains the entire API repsonse in an unmarshalled (`json.RawMessage`) format.
```
Comment (brandtkeller) [inline docs/reference/domains/api-domain.md:58]:
```suggestion
The API response body is serialized into a json object with the `request` `name` as the top-level key. The API status code is included in the output domain resources under `status`. `raw` contains the entire API response in an unmarshalled (`json.RawMessage`) format.
```

**Review by brandtkeller — DISMISSED** [review]
incredibly minor typo otherwise looks great

**Pull request merged**